packages/node/src/lib/log.ts

```diff
@@ -0,0 +1,145 @@
+import type { LogOptions } from './construct-payload';
+import type { GroupingObject, OutgoingLogBody } from './metrics-log';
+import type { Request, Response } from 'express';
+import config from '../config';
+import clamp from 'lodash/clamp';
+import * as url from 'url';
+import { v4 as uuidv4 } from 'uuid';
+import { constructPayload } from './construct-payload';
+import { metricsAPICall } from './metrics-log';
+
+let queue: OutgoingLogBody[] = [];
+function doSend(readmeApiKey, options) {
+  // Copy the queue so we can send all the requests in one batch
+  const json = queue.slice();
```

Comment:
nit: I think it's a little more readable to `[...queue]` if we just want to copy it into a new array

Reply:
c6cd973d3bf4d950b5de9691d37c8b21b29b18e8